src/components/App.js

```diff
@@ -1,14 +1,47 @@
 import React from 'react';
+import Menu from './Menu';
+import Orders from './Orders';
+
 
 class App extends React.Component {
   constructor(){
     super();
+    this.state={update:1};
+    this.updateOrders=this.updateOrders.bind(this);
+    
+  }
+
+  showMenu(event){
+    document.querySelector(".app_menu").style.display="block"
```

Comment:
Avoid using DOM manipulation in React. It would be better to set a value in state and then based on that add a class during render